cmd/policy.go

```diff
@@ -183,12 +184,23 @@ func NewPolicySys() *PolicySys {
 	}
 }
 
-func getConditionValues(request *http.Request, locationConstraint string) map[string][]string {
+func getConditionValues(request *http.Request, locationConstraint string, username string) map[string][]string {
+	currt := UTCNow()
 	args := map[string][]string{
-		"SourceIp":        {handlers.GetSourceIP(request)},
+		"CurrenTime": {currt.Format(event.AMZTimeFormat)},
+		"EpochTime":  {fmt.Sprintf("%d", currt.Unix())},
+		"principaltype": {func() string {
```

Comment:
can this be evaluated before like `UTCNow()` than closure?

Reply:
Sure, I don't know why you prefer one over the other.